standard/requirements/core/REQ_operation.adoc

```diff
@@ -0,0 +1,7 @@
+[[req_core_operation]]
+[width="90%",cols="2,6a"]
+|===
+^|*Recommendation {counter:req-id}* |*/req/core/operation*
+^|A |A WNM SHALL provide the `+properties.operation+` property to indicate if a resource has been updated or deleted.
+^|B |For an update or delete operation, a WNM SHALL provide one link object with `+rel=canonical+`, and `+href=#+`.
```

Comment:
For "delete" yes. The link object is (sort of) empty like that.
For "update", it SHALL provide a new download link.

Reply:
Updated so that empty links are ONLY for delete.